templates/etc/haproxy/backend.cfg.j2

```diff
@@ -23,6 +23,9 @@ backend {{ backend.name }}
 {% if backend.cookie is defined %}
   cookie {{ backend.cookie }}
 {% endif %}
+{% for filter in backend.filter | default([]) %}
+  filter {{ filter.name }}{{ filter.params is defined | ternary([''] + [filter.params], []) | join(' ') }}
+{% endfor %}
```

Comment:
We tend to use something like this:

```
{% for param in server.param | default([]) %} {{ param }}{% endfor %}
```

I find that a bit easier to read.

Reply:
As explained in another of my PR's (#113 ), Ansible has `trim-blocks` enabled, so that format requires the presence of an unexplained blank line after the conditional. the line I used above avoids that.